components/automate-ui/src/app/pages/client-runs/client-runs.component.spec.ts

```diff
@@ -554,10 +546,9 @@ describe('ClientRunsComponent', () => {
         new UpdateNodeFilters({filters: {
           page: 1,
           pageSize: 100,
-          searchBar: [],
+          searchBar: [{type: 'status', text: 'wrong'}],
```

Comment:
I think this should be allowed. This would support future new types of node statuses.